ssh/rootfs/etc/cont-init.d/profile.sh

```diff
@@ -10,6 +10,14 @@ if ! bashio::fs.file_exists /data/.bash_profile; then
     chmod 600 /data/.bash_history
 fi
 
+# Persist tmux configuration by redirecting .tmux.conf to /data
```

Comment:
Thinking about this change a bit more...
If we add this, we can no longer adjust the default configuration for existing installations.
Which will result in new installations behaving differently from existing ones if we need to do that.

@pvizeli what do you think?

Reply:
The reason I picked up the PR is that I did not like the default tmux configuration, and there was no way to update it in a way that would survive a reboot. I guess another option is to put the contents of `tmux.conf` into the addon config?

Reply:
I think it's best to wait with anything like this until <https://github.com/home-assistant/supervisor/issues/1842> is implemented.

Reply:
That indeed looks promising. Should I close this?